cost/azure/unattached_volumes/README.md

```diff
@@ -0,0 +1,56 @@
+# Azure Unused Volumes
+
+## What it does
+
+This Policy Template scans all volumes in the given account and identifies any unattached volumes that have been unattached for at least the number of user-specified days. Using log analytics, we will determine the number of days the disk has been ditached. If any are found, an incident report will show the volumes, and related information and an email will be sent to the user-specified email address.
+
+If the user approves that the volumes should be deleted, the policy will delete the volumes.
+If the volume is not able to be deleted, say, due to it being locked, the volume will be tagged to indicate the error that was received.
+
+If the issue causing the delete failure is removed, the next run of the policy will delete the volume.
+Note: The unattached volumes report will reflect the updated set of unattached volumes on the subsequent run.
+Optionally, the user can specify one or more tags that if found on a volume will exclude the volume from the list.
+
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Unattached days* - The number of days a volume has been unattached.
```

Comment:
make sure this matches the new param label and description

Reply:
Required changes are done